util/cmp/stream.go

```diff
@@ -46,7 +47,7 @@ func ReceiveRepoStream(ctx context.Context, receiver StreamReceiver, destDir str
 	if err != nil {
 		return nil, fmt.Errorf("error receiving tgz file: %w", err)
 	}
-	err = files.Untgz(destDir, tgzFile)
+	err = files.Untgz(destDir, tgzFile, math.MaxInt64)
```

Comment:
Maybe add a comment about why The Limit Does Not Exist here? i.e. we enforce it at the API Server level.

Reply:
it's not enforced anywhere, this is the CMP stream itself which imposes no limits. I kept that behavior by setting the limit to the max.